src/main/java/uk/gov/companieshouse/authcodenotification/controller/AuthCodeNotificationController.java

```diff
@@ -23,11 +23,11 @@
 public class AuthCodeNotificationController {
 
     private final AuthCodeNotificationService authCodeNotificationService;
-    private final DataSanitisation dataSanitisation;
+    private final DataSanitiser dataSanitisation;
```

Comment:
Would rename the variable to match - `dataSanitiser`.

Reply:
updated